src/test/acronym-dialog.html

```diff
@@ -1,154 +0,0 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

Comment:
This used `wrap` and `unwrap` and it also was failing to instantiate properly already.